src/components/navigation/nav.module.scss

```diff
@@ -0,0 +1,317 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,400i,500,500i,700');
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins.scss';
+
+
+.navbar {
+  font-family: $interRegular;
+  font-size: 14px;
+  color: $black;
+  box-sizing: border-box;
```

Comment:
specify the width of the navbar, and give it the width of 100%, as it will be embedded in other components